src/libraries/System.Security.Cryptography.Cose/tests/CoseTestKeyManager.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Test.Cryptography;
+using Xunit;
+
+#pragma warning disable SYSLIB5006
+
+namespace System.Security.Cryptography.Cose.Tests
+{
+    public sealed class CoseTestKeyManager : IDisposable
+    {
+        public const string RSAPkcs1Identifier = "RSA-PKCS1";
+        public const string RSAPssIdentifier = "RSA-PSS";
+        public const string ECDsaIdentifier = "ECDsa";
+        public const string MLDSA44Identifier = "ML-DSA-44";
+        public const string MLDSA65Identifier = "ML-DSA-65";
+        public const string MLDSA87Identifier = "ML-DSA-87";
+
+        private Dictionary<string, CoseTestKey> _coseKeys = new();
+
+        // Ideally public keys should be used with verify, i.e.:
+        // - CoseTestKeyManager.ctor() => private keys
+        // - CoseTestKeyManager CreateTestPublicKeys(CoseTestKeyManager privateKeys)
+        // this is ok for testing purposes.
+        private CoseTestKeyManager() { }
+
+        public static CoseTestKeyManager CreateTestKeys()
+        {
+            CoseTestKeyManager keyManager = new();
+
+            WithTestKeysInfo((keyId, keyType, hashAlgorithm) =>
+            {
+                keyManager.AddKey(keyId, keyType, hashAlgorithm);
+            });
```

Comment:
`keyType` isn't a type of the key, it's a key.  So it should be named appropriately.

```suggestion
            WithTestKeysInfo((keyId, key, hashAlgorithm) =>
            {
                keyManager.AddKey(keyId, key, hashAlgorithm);
            });
```

Reply:
(as a side note hashAlgorithm was completely different type as well here)